pkg/apis/componentconfig/scheme/scheme.go

```diff
@@ -14,24 +14,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package install installs the experimental API group, making it available as
-// an option to all of the API encoding/decoding machinery.
-package install
+package scheme
 
 import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
 	"k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
 )
 
+// Scheme is the runtime.Scheme to which all kubeadm api types are registered.
+var Scheme = runtime.NewScheme()
```

Comment:
I don't think we want this global scheme. This is the wrong direction.

Reply:
Right, we don't want this for the monolithic `componentconfig` API group, but we want it next to the individual API groups. This PR makes `componentconfig` follow that pattern, so it's easy to just move the reference later.